src/test/resources/projects/neoforge/12005/build.gradle

```diff
@@ -0,0 +1,91 @@
+plugins {
+	id 'dev.architectury.loom'
+	id 'maven-publish'
+}
+
+java {
+	sourceCompatibility = JavaVersion.VERSION_21
+	targetCompatibility = JavaVersion.VERSION_21
+}
+
+base {
+	archivesName = project.archives_base_name
+}
+
+version = project.mod_version
+group = project.maven_group
+
+def mcVersion = "@MCVERSION@"
+def neoforgeVersion = "@NEOFORGEVERSION@"
+
+repositories {
+	// Add repositories to retrieve artifacts from in here.
+	// You should only use this when depending on other mods because
+	// Loom adds the essential maven repositories to download Minecraft and libraries from automatically.
+	// See https://docs.gradle.org/current/userguide/declaring_repositories.html
+	// for more information about repositories.
+	maven { url "https://maven.neoforged.net/releases/" }
+	maven {
+		name 'Maven for PR #835' // https://github.com/neoforged/NeoForge/pull/835
+		url 'https://prmaven.neoforged.net/NeoForge/pr835'
```

Comment:
Are these PR repos future-proof? I don't want the tests to suddenly start failing 😅